data/wanderer ships.txt

```diff
@@ -562,7 +566,8 @@ ship "Deep River"
 			"hit force" 4200
 	outfits
 		"Thunderhead Launcher" 6
-		"Thunderhead Missile" 240
+		"Thunderhead Missile" 320
+		"Thunderhead Storage Array" 4
```

Comment:
thinking about removing one for the sake of OCD: the last volley has 2 missiles that shoot when it has 6 launchers
it would also pair with balancing a bit